rubin_sim/maf/metrics/visit_groups_metric.py

```diff
@@ -65,7 +69,8 @@ def run(self, data_slice, slice_point=None):
 
 
 class VisitGroupsMetric(BaseMetric):
-    """Count the number of visits per night within delta_t_min and delta_t_max."""
+    """Count the number of visits per night within delta_t_min
+    and delta_t_max."""
```

Comment:
This matches ruff, but since you're here editing ... 
the docstring after the "__init__" below with the parameters should come up to this section of documentation as Parameters and reformat into numpydoc style.

Reply:
would it be okay to leave this update for the issue focused on the developer guide?

Reply:
Yes, ok.